Bottom: 070f92a593aea528dcae7a68d42159b016d77109
Top:    a069666dc428f246f9dc954bdb50553367794189
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-17 12:17:16 -0700

Refresh of rs-fulltest

---

diff --git a/examples/rstream.c b/examples/rstream.c
index cb65911..104b318 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -154,16 +154,16 @@ static void show_perf(void)
 	bytes = (long long) iterations * transfer_count * transfer_size * 2;
 
 	/* name size transfers iterations bytes seconds Gb/sec usec/xfer */
-	printf("%s\t", test_name);
+	printf("%-10s", test_name);
 	size_str(str, transfer_size);
-	printf("%s\t", str);
+	printf("%-8s", str);
 	cnt_str(str, transfer_count);
-	printf("%s\t", str);
+	printf("%-8s", str);
 	cnt_str(str, iterations);
-	printf("%s\t", str);
+	printf("%-8s", str);
 	size_str(str, bytes);
-	printf("%s\t", str);
-	printf("%.2fs \t%.2f \t%.2f\n",
+	printf("%-8s", str);
+	printf("%8.2fs%10.2f%11.2f\n",
 		usec / 1000000., (bytes * 8) / (1000. * usec),
 		(usec / iterations) / (transfer_count * 2));
 }
@@ -504,7 +504,8 @@ static int run(void)
 		goto free;
 	}
 
-	printf("name \tbytes \txfers \titers \ttotal \ttime \tGb/sec \tusec/xfer\n");
+	printf("%-10s%-8s%-8s%-8s%-8s%8s %10s%13s\n",
+	       "name", "bytes", "xfers", "iters", "total", "time", "Gb/sec", "usec/xfer");
 	if (!custom) {
 		for (i = 0; i < TEST_CNT; i++) {
 			if (test_size[i].option > size_option)
